app/main.py

```diff
@@ -2,0 +1,10 @@
+def copy_file(command: str) -> None:
+    command_list = command.split()
+    file_name = command_list[1]
+    new_file_name = command_list[-1]
```

Comment:
The variable `new_file_name` could be more descriptive. Consider renaming it to something like `destination_file_name` for better readability, as suggested in the checklist.